### PR TITLE
edb can backup only selected tables or without some tables, switch on…

### PR DESCRIPTION
**Pull request by brookzhang**
… backup all cluster or not

These changes are for my requirement, please check if they're valuable enough to merge into your gem.

### CODE REVIEW

File: example/edb.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@
     :database: sample_database
     :host: localhost
     :port: 5432
+    :only_tables: ['table1', 'table2']
```
Comment (RoxasShadow) [inline example/edb.yml]: I suggest to give this option an empty array to be coherent with `without_tables`.

File: example/edb.yml [hunk 2]
```diff
@@ -7,6 +7,10 @@
...
+    :without_tables: ['table1', 'table2']
```
Comment (RoxasShadow) [inline example/edb.yml]: I suggest to rename this option to `exclude_tables` to match the name used by PostgreSQL.

File: example/edb.yml [hunk 3]
```diff
@@ -7,6 +7,10 @@
...
+    :run_sql_after_backup: ~/Documents/clear_some_tables_after_backup.sql
```
Comment (RoxasShadow) [inline example/edb.yml]: This is never used inside your code (and I honestly think this is not a good place to put that inside the PostgreSQL block).

File: example/edb.yml [hunk 4]
```diff
@@ -7,6 +7,10 @@
...
+    :backup_all_cluster: true
```
Comment (RoxasShadow) [inline example/edb.yml]: What do you think about `include_cluster`?

**Comment by RoxasShadow** [comment]
Hello, thank you for your PR.

I left you some comments, please have a look when you can. Thanks.

**Comment by brookzhang** [comment]
Cool comments!  Will fix it soon.

**Comment by brookzhang** [comment]
Hello Roxas,
I've finished all changes. For run_sql_after_backup which won't be used for most user , its upon you to merge it or not.   
Thanks.

**Comment by RoxasShadow** [comment]
Great, I'm going to merge this. Thanks.

**Pull request merged**